src/test/java/org/aksw/gerbil/annotator/TestAnnotatorConfiguration.java

```diff
@@ -37,7 +37,7 @@ public TestAnnotatorConfiguration(List<Document> instances, ExperimentType appli
 
     public TestAnnotatorConfiguration(String annotatorName, boolean couldBeCached, List<Document> instances,
             ExperimentType applicableForExperiment) {
-        super(annotatorName, couldBeCached, applicableForExperiment);
+        super(annotatorName,"Un Grouped", couldBeCached, applicableForExperiment);
```

Comment:
Please use the static attribute that you defined further above.

Reply:
Done